lighthouse-core/audits/user-timings.js

```diff
@@ -118,6 +118,7 @@ class UserTimings extends Audit {
         const time = item.isMark ? item.startTime : item.duration;
         return {
           name: item.name,
+          startTime: item.startTime,
           timingType: item.isMark ? 'Mark' : 'Measure',
           time,
```

Comment:
just noticed.. this name seems pretty weak. duration? timeInMs?

Reply:
well that name is weak because it's either or and just for display, I added duration separately, but programmatic folks should probably just ignore `time`

maybe update to `timeToDisplay`?

Reply:
it's always been a bit odd that we share a column for two diff types of values

how about we add a second column, then we can be really clear with `Start time` and `Duration`. wdyt?